### DISCUSSION TITLE
Antiburn on Lanbon L8

### DISCUSSION THREAD

**Discussion by dodacs** [HASwitchPlate/openHASP#360] [Q&A]
I have the Lanbon L8 installed horizontally (90 degrees on display option) and when I run the anti burn option from the home assistant integration, it only runs on part of the screen.  (like the coordinates are for vertical screen). 
Where can I change the coordinates fro the antiburn function?

**Comment by fvanroie** [comment]
This issue should already be fixed in 0.7.0-rc https://github.com/HASwitchPlate/openHASP/issues/389
